core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -15,10 +15,15 @@
 import alluxio.Configuration;
 import alluxio.Constants;
 import alluxio.collections.Pair;
+import alluxio.security.authorization.PermissionStatus;
+import alluxio.underfs.options.UnderFileSystemCreateOptions;
```

Comment:
updated. I was trying to distinguish UFS op option names from Alluxio op options.